packages/suite/src/views/dashboard/AssetsView/AssetTable/AssetStakingRow.tsx

```diff
@@ -0,0 +1,46 @@
+import { Column, Icon, Table, Text } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+import { CoinBalance, HiddenPlaceholder, Translation, FiatValue } from 'src/components/suite';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { AssetTableExtraRowsSection as Section } from './AssetTableExtraRowsSection';
+
+interface AssetStakingRowProps {
+    stakingTotalBalance: string;
+    symbol: NetworkSymbol;
+    renderBothRows: boolean;
```

Comment:
`shouldRenderBothRows`

Reply:
or maybe `shouldRenderTokenRow`? as you know that you should render staking at that place

Reply:
Yes, good catch https://github.com/trezor/trezor-suite/pull/14996/commits/5a47ab41b461e17bd4f7eeafdfd91ebfc2884ae7